packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -361,15 +361,21 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 	/**
 	 * Error raising for socket.io issues
 	 */
-	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
-	protected createErrorObject(handler: string, error?: any, canRetry = true): IAnyDriverError {
+	protected createErrorObject(
+		socketEvent: string,
+		// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+		error?: any,
+		canRetry = true,
+	): IAnyDriverError {
 		// Note: we suspect the incoming error object is either:
 		// - a socketError: add it to the OdspError object for driver to be able to parse it and reason over it.
 		// - anything else: let base class handle it
-		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-		return canRetry && Number.isInteger(error?.code) && typeof error?.message === "string"
-			? errorObjectFromSocketError(error as IOdspSocketError, handler)
-			: super.createErrorObject(handler, error, canRetry);
+		const errorObject =
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+			canRetry && Number.isInteger(error?.code) && typeof error?.message === "string"
+				? errorObjectFromSocketError(error as IOdspSocketError, socketEvent)
+				: super.createErrorObject(socketEvent, error, canRetry);
```

Comment:
I might be wrong, but I think we are doing the same mistake here. We are not adding "errorFrom" in flow of "super.createErrorObject". I saw that you added "errorFrom" only at 1 place which is in "errorObjectFromSocketError" and not in base class.

Reply:
That was intentional, since originally I was making the change only limited to odsp-driver. I have updated the code to now have `eventName` property on the `IAnyDriverError` interface, and made it more generic. So now the code updates this property both at base as well as odsp level. Please review now.